website/snippets/_dbt-package-hub-env-var.md

```diff
@@ -0,0 +1,22 @@
+If you're hosting an internal version of the [dbt Labs hosted package hub](https://hub.getdbt.com/), you can set the [`DBT_PACKAGE_HUB_URL`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/clients/registry.py#L23) environment variable to point to your instance of the dbt package hub. This allows you to install packages from your self-hosted version. 
```

Comment:
should i add this at the end: "This allows you to install packages from your self-hosted version and not the public hub version.."